packages/@aws-cdk/tmp-toolkit-helpers/src/api/io/private/io-helper.ts

```diff
@@ -13,7 +13,7 @@ export type ActionLessRequest<T, U> = Omit<IoRequest<T, U>, 'action'>;
  * Wraps a client provided IoHost and provides additional features & services to toolkit internal classes.
  */
 export interface IoHelper extends IIoHost {
-  notify<T>(msg: ActionLessMessage<T>): Promise<void>;
+  notify(msg: ActionLessMessage<any>): Promise<void>;
```

Comment:
This feels bad. Why are you are going against your own advice from the PR description?
```suggestion
  notify(msg: ActionLessMessage<unkown>): Promise<void>;
```

Reply:
Hah. Because of this:

<img width="1248" alt="image" src="https://github.com/user-attachments/assets/7a1f1335-c6e3-4fc7-9b65-fe045a3d182e" />

Ideally we'd do something like:

```ts
IO.CDK_TOOLKIT_I5501.getData(msg)
```

Or something

Reply:
I had thought that TypeScript would treat an unbound `<T>` as unknown and force you to test. But looks like it just treats it like `any` and says "you know what you're doing". It's not like we used to have type safety that we've now lost